crates/sui-rpc-api/proto/sui/rpc/v2alpha/bcs.proto

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package sui.rpc.v2alpha;
+
+// `Bcs` contains an arbitrary type that is serialized using the
+// [BCS](https://mystenlabs.github.io/sui-rust-sdk/sui_sdk_types/index.html#bcs)
+// format as well as a name that identifies the type of the serialized value.
+message Bcs {
+  // Name that identifies the type of the serialized value.
+  optional string name = 1;
```

Comment:
This is an interesting one. What will this be used for and what's the convention for these names? (In particular, what name would we give for the contents of serialized move values?)